src/scancode/cli.py

```diff
@@ -439,13 +439,9 @@ def scancode(
         cliutils.validate_option_dependencies(ctx)
         pretty_params = get_pretty_params(ctx, generic_paths=test_mode)
 
-        # warn for outdated version and/or check for updates
-        from scancode.outdated import check_scancode_version_locally
-        outdated = check_scancode_version_locally()
-
-        if not outdated and check_version:
-            from scancode.outdated import check_scancode_version_remotely
-            outdated = check_scancode_version_remotely()
+        # Check for updates
```

Comment:
This should ONLY happen `if check_version`. This way folks can elect to not do any such check